### PR TITLE
[hi] Add  content/hi/docs/tasks/tools/included/optional-kubectl-confi…

### PR DESCRIPTION
**Pull request by ashish-jaiswar**
### Added one file
localize optional-kubectl-configs-bash-linux.md to Hindi.
[issue](https://github.com/kubernetes/website/issues/30569)

### CODE REVIEW

**Comment by ashish-jaiswar** [comment]
hey @anubha-v-ardhan can you review this PR ?

File: content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+title: "Linux पर bash ऑटो-कम्पलीशन"
+description: "Linux पर bash ऑटो-कम्पलीशन के लिए कुछ वैकल्पिक विन्विन्यास।"
+headless: true
+---
+
+## परिचय
+
+Bash के लिए kubectl  कम्प्लिशन स्क्रिप्ट `kubectl completion bash` कमांड के साथ उत्पन्न की जा सकती है। आपके शेल में कम्प्लिशन स्क्रिप्ट को सोर्स करने से kubectl ऑटोकम्प्लिशन ऐनेबल हो जाती है।
```
Comment (Garima-Negi) [inline content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md:9]: We'd want to use a fixed spelling for `completion` : here we have used `कम्पलीशन` and `कम्प्लिशन` both. I suggest going forward with `कम्पलीशन`

**Review by Garima-Negi — CHANGES_REQUESTED** [review]

**Comment by ashish-jaiswar** [comment]
@Garima-Negi, @anubha-v-ardhan   I have updated the file, can you review this file?

File: content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+---
+title: "Linux पर bash ऑटो-कम्पलीशन"
+description: "Linux पर bash ऑटो-कम्पलीशन के लिए कुछ वैकल्पिक विन्विन्यास।"
+headless: true
+---
+
+## परिचय
+
+Bash के लिए kubectl  कम्पलीशन स्क्रिप्ट `kubectl completion bash` कमांड के साथ उत्पन्न की जा सकती है। आपके शेल में कम्पलीशन स्क्रिप्ट को सोर्स करने से kubectl ऑटोकम्पलीशन ऐनेबल हो जाती है।
```
Comment (divya-mohan0209) [inline content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md:9]: I believe कम्पलीशन can be replaced by समापन? Or are we trying not to stray too far away from the terminology that is being introduced?

Comment (divya-mohan0209) [inline content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md:9]: Also ऐनेबल can be replaced by सक्षम करना

Comment (divya-mohan0209) [inline content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md:9]: Extra space between kubectl and कम्पलीशन

File: content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+हालाँकि, कम्पलीशन की स्क्रिप्ट [**bash-completion**](https://github.com/scop/bash-completion) पर निर्भर करती हैं जिसका अर्थ है कि आपको पहले इस सॉफ़्टवेयर को स्थापित करना होगा (आप `type _init_completion`  चलाकर परीक्षण कर सकते हैं कि आपने  पहले से bash-completion इंस्टॉल की है या नहीं)।
```
Comment (divya-mohan0209) [inline content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md:11]: > हालाँकि, कम्पलीशन की स्क्रिप्ट [**bash-completion**](https://github.com/scop/bash-completion) पर निर्भर करती हैं

Can be rephrased for grammatical correctness as below

हालाँकि, कम्पलीशन की स्क्रिप्ट [**bash-completion**](https://github.com/scop/bash-completion) पर निर्भर हैं

Comment (divya-mohan0209) [inline content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md:11]: There are multiple double spaces in the second line. Please amend.

File: content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+## Bash-completion को इंस्टॉल करें
+
+कई पैकेज मैनेजर द्वारा bash-completion प्रदान की जाती है ([यहाँ](https://github.com/scop/bash-completion#installation) देखें)। आप इसे `apt-get install bash-completion` या `yum install bash-completion` आदि के साथ इंस्टॉल कर सकते हैं।
```
Comment (divya-mohan0209) [inline content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md:15]: We need to use either इंस्टॉल or स्थापित. Using both of them will be unnecessary across the document. Please make amends.

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]
PTAL

File: content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md (hunk 2 above)
Comment (ashish-jaiswar) [reply content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md:9]: if I replace  कम्पलीशन by समापन, then what about this auto-completion, bash-completion ?

**Comment by ashish-jaiswar** [comment]
@divya-mohan0209 , @Garima-Negi , @anubha-v-ardhan I have updated the file, Can I squash the commit?

**Comment by ashish-jaiswar** [comment]
@divya-mohan0209 can you merge this PR? I have updated this file.

File: content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+हालाँकि, समापन की स्क्रिप्ट [**bash-completion**](https://github.com/scop/bash-completion) पर निर्भर हैं जिसका अर्थ है कि आपको पहले इस सॉफ़्टवेयर को इंस्टॉल करना होगा (आप `type _init_completion`  चलाकर परीक्षण कर सकते हैं कि आपने पहले से bash-completion इंस्टॉल की है या नहीं)।
+
+## Bash-completion को इंस्टॉल करें
+
+कई पैकेज मैनेजर द्वारा bash-completion प्रदान की जाती है ([यहाँ](https://github.com/scop/bash-completion#installation) देखें)। आप इसे `apt-get install bash-completion` या `yum install bash-completion` आदि के साथ इंस्टॉल कर सकते हैं।
+
+यह कमांड `/usr/share/bash-completion/bash_completion` उत्त्पन्न करता है, जो bash-completion की मुख्य स्क्रिप्ट है। आपके पैकेज मैनेजर के आधार पर, आपको इस फाइल को अपनी `~/.bashrc` फाइल में मैन्युअल रूप से सोर्स करना होगा।
+
+यह पता लगाने के लिए, अपना शेल पुनः लोड करें और `type _init_completion` रन करे। यदि कमांड सफल होता है, तो आप पहले से ही सेट हैं, अन्यथा अपनी `~/.bashrc` फ़ाइल में निम्नलिखित जोड़ें:
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md:19]:
```suggestion
यह पता लगाने के लिए, अपना शेल पुनः लोड करें और `type _init_completion` रन करे। यदि कमांड सफल होता है, तो आप पहले से ही तैयार हैं, अन्यथा अपनी `~/.bashrc` फ़ाइल में निम्नलिखित जोड़ें:

```

File: content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+```bash
+source /usr/share/bash-completion/bash_completion
+```
+
+अपना शेल पुनः लोड करें और `type _init_completion` टाइप करके सत्यापित करें कि bash-completion सही ढंग से इंस्टॉल है।
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md:24]:
```suggestion
अपना शेल पुनः लोड करें और `type _init_completion` टाइप करके सत्यापित करें कि बैश-कम्पलीशन सही ढंग से इंस्टॉल है।
```

File: content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+
+## kubectl ऑटोकम्पलीशन सक्षम करे
+
+अब आपको यह सुनिश्चित करने की आवश्यकता है कि Kubectl समापन स्क्रिप्ट आपके सभी शेल सत्रों (sourced) में प्राप्त हो जाए। आप ऐसा दो तरीकों से कर सकते हैं:
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md:28]:
```suggestion
अब आपको यह सुनिश्चित करने की आवश्यकता है कि kubectl समापन स्क्रिप्ट आपके सभी शेल सत्रों (sourced) में प्राप्त हो जाए। आप ऐसा दो तरीकों से कर सकते हैं:
```

File: content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+   ```bash
+  echo 'source <(kubectl completion bash)' >>~/.bashr
+  ```
+ -   समापन स्क्रिप्ट को `/etc/bash_completion.d` डायरेक्टरी में जोड़ें:
+    ```bash
+    kubectl completion bash >/etc/bash_completion.d/kubectl
+    ```
+    
+
+यदि आप के पास kubectl के लिए एक अन्य नाम (alias) है, तो आप उस अन्य नाम के साथ काम करने के लिए शेल समापन को बढ़ा सकते हैं:
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md:41]:
```suggestion
यदि आप के पास kubectl के लिए एक अन्य नाम (alias) है, तो आप उस अन्य नाम के साथ काम करने के लिए शेल समापन को बढ़ा सकते हैं:

```

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]
some NITS, LGTM otherwise

File: content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md [hunk 9]
```diff
@@ -0,0 +1,50 @@
+---
+title: "Linux पर bash ऑटो-कम्पलीशन"
+description: "Linux पर bash ऑटो-कम्पलीशन के लिए कुछ वैकल्पिक विन्विन्यास।"
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md:3]:
```suggestion
description: "Linux पर bash ऑटो-कम्पलीशन के लिए कुछ वैकल्पिक कॉन्फ़िगरेशन।"
```

Comment (anubha-v-ardhan) [reply content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md:3]: We have used कॉन्फ़िगरेशन everywhere for easy understanding. I guess we should maintain that here too

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]

**Comment by anubha-v-ardhan** [comment]
/label tide/merge-method-squash
/assign

@ashish-jaiswar If you know how to squash commits down to 1 commit, Can you do it? Its okay if you don't too.

**Comment by ashish-jaiswar** [comment]
@anubha-v-ardhan  I will squash the commit and merge it into a single commit.

**Comment by ashish-jaiswar** [comment]
@anubha-v-ardhan can you squash the commit?

**Comment by anubha-v-ardhan** [comment]
/lgtm
/approve

Thank you @ashish-jaiswar

**Pull request merged**